### PR TITLE
Added some clarifying statements to readmes

### LINKED ISSUE #8628: Clarify REAME for Azure Storage Blob client
**Query/Question**

Please add a description of what people are supposed to put in `<your-storage-account-url>` mentioned multiple times in [Azure Storage Blob client README](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-blob).

It is mentioned multiple times throughout the example, but the value people expected to put there is never described. Found it in the [migration guide](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-blob/migrationGuides/V8_V12.md) and it is absolutely not obvious, so would be good to add a section on where to get it, similar to how access token is described.

***Why is this not a Bug or a feature Request?***
It is a documentation improvement request, not a bug or a feature for the library itself.

**Setup (please complete the following information if applicable):**
Not applicable.
 
 **Information Checklist**
 Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [x] Query Added
- [x] Setup information Added

### PR DESCRIPTION
**Pull request by gapra-msft**
Resolves #8628

### CODE REVIEW

File: sdk/storage/azure-storage-blob/README.md [hunk 1]
```diff
@@ -105,6 +108,30 @@ Blob Storage is designed for:
...
+```$xslt
+https://myaccount.blob.core.windows.net/mycontainer
+```
+
+For a blob, the base URI includes the name of the account, the name of the container and the name of the blob:
+
+```$xslt
+https://myaccount.blob.core.windows.net/mycontainer/myblob
+```
+
```
Comment (rickle-msft) [inline sdk/storage/azure-storage-blob/README.md]: Maybe add a note that CDNs or other specific scenarios my have different formats? That could be too much for the README, though.

File: sdk/storage/azure-storage-blob/README.md [hunk 2]
```diff
@@ -105,6 +108,30 @@ Blob Storage is designed for:
...
 
+### URL format
+Blobs are addressable using the following URL format:
+The following URL addresses a blob:
+https://myaccount.blob.core.windows.net/mycontainer/myblob
+
+#### Resource URI Syntax
+For the storage account, the base URI for blob operations includes the name of the account only:
+
+```$xslt
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/README.md]: Does that refer to https://en.wikipedia.org/wiki/XSLT ? I wonder why such choice here.

File: sdk/storage/azure-storage-blob/README.md (hunk 1 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/README.md]: yeah I'm thinking this is more for people using this for the first time? for more complex scenarios I'm assuming people will know what theyre doing

File: sdk/storage/azure-storage-blob/README.md (hunk 2 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/README.md]: tbh I copied it from the queue README

Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/README.md]: This is what it ends up generating as (take a look at the url part of the README) https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-queue

Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob/README.md]: It doesn't seem to bring any special formatting when I see preview of this page. Why don't we drop it?
It brings back some bad memories from the past.

Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/README.md]: sure, I'll remove it

Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/README.md]: removed!

File: sdk/storage/azure-storage-blob/README.md (hunk 1 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/README.md]: just added a note that this doesnt apply for more advanced scenarios like custom domain names

**Pull request merged**